src/index.ts

```diff
@@ -27,7 +30,7 @@ const type = (input: HTMLElement, text: string) => {
 const clear = async (input: HTMLElement, clearButton: Element) => {
   fireEvent.mouseDown(clearButton);
   fireEvent.click(clearButton);
-  // react-select will prevent the menu from opening, and asynchronously focus the select field...
+  // react-select will prevent the menu from opening, and asynchronously blur the select field...
```

Comment:
I believe this comment was correct - react-select focuses the input field after a clear.
We need to wait for this focus and then blur the field again to leave everything back in the initial state.

Reply:
Ah gotcha, I saw the `blur` below and thought you were talking about that.